### DISCUSSION TITLE
How to get snaps of good image quality or resolution

### DISCUSSION THREAD

**Discussion by arunnbabu81** [buggyj/tiddlyclip-plugin#92] [Q&A]
Is there anyway to control the image quality or resolution of snaps in the tiddlyclipconfig

**Comment by buggyj** [comment]
Snaps are screen shots so have the resolution of you screen. If you zoom the image to the maximum size of you screen (shift with +) Then the image will be larger and therefore have more resolution, BUT it will appear larger in your tiddlywiki unless you control it - eg 

`[img width=32 [mysnap.png]]`
